frontend/src/app/model/ChainSelection.ts

```diff
@@ -3,42 +3,26 @@ import {Point} from './Point';
 
 export class ChainSelection extends SliceSelection {
 
-    private _points: Array<Point>;
-    private _isLoop: boolean;
+    points: Array<Point>;
```

Comment:
The problem is that part of models use public variables (e.g. Label, LabelTag, UserInfo) and part of them use private variables (e.g. ROISelection2D, SliceRequest). Unfortunately, we don't have consistency. I think that models should have public variables because they don't have any business logic and services should have private variables + getters/setters.

Rest of the team should give its opinion.

Reply:
Yes, there are a lot of inconsistent places in Frontend... We should talk about code quality at Wednesday during our Retrospective & Planning meeting.

For now, let's ask @Darkoon for his opinion.

Reply:
It really isn't binary choice here. We should not go "all public" or "all private" as a matter of "consistency" but go with better flow of information between frontend modules. I think that model classes should be as closed as possible to avoid near limitless object transformation that buggs so many js codebases. Services should be open when it comes to its "domain" functions with closed core implementation. As our code is now more dynamic than ever, there are some cleanup tasks to be done in future (code cleanup, maybe some more structural changes, scan-viewer future, tests etc), but I think that we will do it when our "core" functionalities will be done (I will do a full sweep of frontend codebase and add necessary tasks to our backlog). In cases like this, please consider main idea for class. Model classes like that one should deliver some kind of confident info from component to component. Give another develops power to change something only in contexts that you find "secure".